GliaWidgets/SecureConversations/SecureConversations.MessagesWithUnreadCountLoader.swift

```diff
@@ -0,0 +1,91 @@
+extension SecureConversations {
```

Comment:
do we need to add `import Foundation` for SPM support?

Reply:
Good catch. Added it e20b1b5aaad07503d4989e9de5aef9f0d165c4f4.